packages/ui/client/components/FileDetails.vue

```diff
@@ -19,27 +19,29 @@ const graph = useModuleGraph(data)
 </script>
 
 <template>
-  <div v-if="current" h-full w-full overflow="hidden">
-    <div p="2" h-10 flex="~ gap-2" items-center bg-header border="b base">
-      <StatusIcon :task="current" />
-      <div flex-1 font-light op-50 ws-nowrap tuncate text-sm>
-        {{ current?.filepath }}
+  <div v-if="current" h-full>
+    <div scroll-header>
+      <div p="2" h-10 flex="~ gap-2" items-center bg-header border="b base">
+        <StatusIcon :task="current" />
+        <div flex-1 font-light op-50 ws-nowrap truncate text-sm>
+          {{ current?.filepath }}
+        </div>
+        <div class="flex text-lg">
+          <IconButton icon="i-carbon-launch" :disabled="!current?.filepath" :onclick="open" />
+        </div>
       </div>
-      <div class="flex text-lg">
-        <IconButton icon="i-carbon-launch" :disabled="!current?.filepath" :onclick="open" />
+      <div flex="~" items-center bg-header border="b base" text-sm h-37px>
+        <button tab-button :class="{ 'tab-button-active': viewMode == null }" @click="viewMode = null">
```

Comment:
I suggest you create a method that change the view state I.e
const changeViewMode = (view) => viewMode = view.

<button @click="changeViewMode('graph')"></button>

Reply:
@dammy001 changed